semant.ml

```diff
@@ -57,8 +62,14 @@ let check (globals, functions) =
        (* change formals to be variadic? Right now, this is fixed by just not 
        comparing formals and actuals list if the name of the function is printf  *)
        locals = []; body = [] });
+
        ("print_stone", { typ = Int; fname = "print_stone"; formals = [(Stone,
-       "x")]; locals = []; body = [] })] 
+       "x")]; locals = []; body = [] });
+
+       ("scanf", { typ = Void; fname = "scanf"; formals = [(Pointer(Char), "x")]; locals = []; body = [] });
```

Comment:
What else besides chars would you want to malloc? I'm pretty sure nothing. We don't have structs or anything like that. 
Obv this is weird but def not a huge issue